src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -458,6 +458,17 @@ public bool IsIdentity
         }
 
         /// <summary>
+        /// Returns true if the conversion is a stackalloc conversion.
+        /// </summary>
+        public bool IsStackAlloc
```

Comment:
For other conversion kinds, the public API allows you to figure out exactly which kind it is. For example, if it has IsNumeric and IsImplicit, then you know it was ConversionKind.ImplicitNumeric.
The public API for stack alloc doesn't allow you to figure out which one it was.
Can you add a work item or PROTOTYPE to review with Neal?

Reply:
Is this property tested?

---
In reply to: [136683924](https://github.com/dotnet/roslyn/pull/21746#discussion_r136683924) [](ancestors = 136683924)

Reply:
@jcouv The public api lets you know which one it was by testing the resulting type for being a pointer type.

Reply:
Done.

---
In reply to: [136685033](https://github.com/dotnet/roslyn/pull/21746#discussion_r136685033) [](ancestors = 136685033,136683924)